spec/views/exports/hamilton_lrc_pbmc_bank_to_cellaca_common.csv.erb_spec.rb

```diff
@@ -31,65 +31,44 @@
 
   let(:workflow) { 'scRNA Core LRC PBMC Bank Cell Count' }
 
-  let(:template) { "exports/hamilton_lrc_pbmc_bank_to_cellaca_max_#{count}_count" }
-
   before do
     assign(:plate, plate)
     assign(:workflow, workflow)
   end
 
-  context 'with hamilton_lrc_pbmc_bank_to_cellaca_max_4_count' do
-    let(:count) { 4 }
-    it 'renders the expected content' do
-      render(template: template)
-      content = CSV.parse(rendered)
-      expect(content).to eq(expected_content(count))
-    end
-  end
-
-  context 'with hamilton_lrc_pbmc_bank_to_cellaca_max_6_count' do
-    let(:count) { 6 }
-    it 'renders the expected content' do
-      render(template: template)
-      content = CSV.parse(rendered)
-      expect(content).to eq(expected_content(count))
-    end
-  end
-
-  context 'with hamilton_lrc_pbmc_bank_to_cellaca_max_12_count' do
-    let(:count) { 6 }
+  context 'with first replicates' do
+    let(:template) { 'exports/hamilton_lrc_pbmc_bank_to_cellaca_first_count' }
     it 'renders the expected content' do
       render(template: template)
       content = CSV.parse(rendered)
-      expect(content).to eq(expected_content(count))
+      selected_wells = Utility::CellCountSpotChecking.new(plate).first_replicates
+      expect(content).to eq(expected_content(selected_wells))
     end
   end
 
-  context 'with hamilton_lrc_pbmc_bank_to_cellaca_all_count' do
-    let(:count) { nil }
-    let(:template) { 'exports/hamilton_lrc_pbmc_bank_to_cellaca_all_count' }
+  context 'with second replicates' do
+    let(:template) { 'exports/hamilton_lrc_pbmc_bank_to_cellaca_second_count' }
     it 'renders the expected content' do
       render(template: template)
       content = CSV.parse(rendered)
-      expect(content).to eq(expected_content(count))
+      selected_wells = Utility::CellCountSpotChecking.new(plate).second_replicates
+      expect(content).to eq(expected_content(selected_wells))
     end
   end
 
```

Comment:
maybe add a test including one where there is no second replicate? (if this is appropriate)

Reply:
Thanks. I will add a test for this case.